source/lib/src/fmt_nlist.cc

```diff
@@ -1,246 +1,185 @@
-    const float &		rcut,
-    const std::vector<int > &   sec_a)
-{
-    fmt_nei_idx_a.resize (sec_a.back());
-    fill (fmt_nei_idx_a.begin(), fmt_nei_idx_a.end(), -1);
-  
-    // gether all neighbors
-    std::vector<int > nei_idx (nei_idx_a);
-    // allocate the information for all neighbors
-    std::vector<NeighborInfo<float> > sel_nei;
-    sel_nei.reserve (nei_idx_a.size());
-    float rcut2 = rcut * rcut;
-    for (unsigned kk = 0; kk < nei_idx.size(); ++kk) {
-        // rcut is float in this function, so float rr is enough
-        float diff[3];
-        const int & j_idx = nei_idx[kk];
-        for (int dd = 0; dd < 3; ++dd) {
-            diff[dd] = (float)posi[j_idx * 3 + dd] - (float)posi[i_idx * 3 + dd];
-        }
-        float rr2 = deepmd::dot3(diff, diff);    
-        if (rr2 <= rcut2) {
-            sel_nei.push_back(NeighborInfo<float>(type[j_idx], rr2, j_idx));
-        }
+  // gether all neighbors
+  std::vector<int> nei_idx(nei_idx_a);
+  // allocate the information for all neighbors
+  std::vector<NeighborInfo<float>> sel_nei;
+  sel_nei.reserve(nei_idx_a.size());
+  float rcut2 = rcut * rcut;
+  for (unsigned kk = 0; kk < nei_idx.size(); ++kk) {
+    // rcut is float in this function, so float rr is enough
+    float diff[3];
+    const int &j_idx = nei_idx[kk];
+    for (int dd = 0; dd < 3; ++dd) {
+      diff[dd] = (float)posi[j_idx * 3 + dd] - (float)posi[i_idx * 3 + dd];
     }
-    sort(sel_nei.begin(), sel_nei.end());  
-  
-    std::vector<int > nei_iter = sec_a;
-    int overflowed = -1;
-    for (unsigned kk = 0; kk < sel_nei.size(); ++kk) {
-        const int & nei_type = sel_nei[kk].type;
-        if (nei_iter[nei_type] < sec_a[nei_type+1]) {
-            fmt_nei_idx_a[nei_iter[nei_type] ++] = sel_nei[kk].index;
-        }
-	else{
-	  overflowed = nei_type;
-	}
+    float rr2 = deepmd::dot3(diff, diff);
+    if (rr2 <= rcut2) {
+      sel_nei.push_back(NeighborInfo<float>(type[j_idx], rr2, j_idx));
+    }
+  }
+  sort(sel_nei.begin(), sel_nei.end());
+
+  std::vector<int> nei_iter = sec_a;
+  int overflowed = -1;
+  for (unsigned kk = 0; kk < sel_nei.size(); ++kk) {
+    const int &nei_type = sel_nei[kk].type;
+    if (nei_iter[nei_type] < sec_a[nei_type + 1]) {
+      fmt_nei_idx_a[nei_iter[nei_type]++] = sel_nei[kk].index;
+    } else {
+      overflowed = nei_type;
     }
-    return overflowed;
+  }
+  return overflowed;
 }
 
-template<typename FPTYPE> 
-void 
-deepmd::
-format_nlist_cpu (
-    int * nlist,
-    const InputNlist & in_nlist,
-    const FPTYPE * coord, 
-    const int * type, 
-    const int nloc, 
-    const int nall, 
-    const float rcut, 
-    const std::vector<int> sec)
-{
+template <typename FPTYPE>
+void deepmd::format_nlist_cpu(int *nlist, const InputNlist &in_nlist,
+                              const FPTYPE *coord, const int *type,
+                              const int nloc, const int nall, const float rcut,
+                              const std::vector<int> sec) {
```

Comment:
Grouping some of the parameters in one line may not be a good idea. 
I would prefer the old style: one parameter a line.

Reply:
[`BinPackParameters`](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#binpackparameters) can control it.

Reply:
@njzjz please try the google's style

Reply:
Done. Please check 2baf9f43ce6b10ecc0a784ccb1528fbaaad31042

Reply:
It looks like we still need to set `BinPackParameters`.

Reply:
890365d2d5d2c21dea37e1cfd76d0a3d3e055195